GameFramework/RotaionValue/RotationValue.h

```diff
@@ -0,0 +1,65 @@
+﻿//----------------------------------------------------------
+// <filename>RotationValue.h</filename>
+//----------------------------------------------------------
+
+#ifndef ROTATION_VALUE_H
+#define ROTATION_VALUE_H
+
+#include <Window.h>
+
+#include <d3dx9.h>
+
+/// <summary>
+/// 基礎構築をに関するものをまとめた名前空間
+/// </summary>
+namespace gameframework
+{	
+	/// <summary>
+	/// 角度の基底クラス
+	/// </summary>
+	class RotationValue
+	{
+	public:
+		RotationValue(float amount, float min, float oneCycleAmount) :ONE_CYCLE_AMOUNT(oneCycleAmount)
+		{
+			m_rawAmount = amount;
+
+			Normalize(amount, min);
+		}
+
+		virtual ~RotationValue() =0;
+
+		inline float Raw() const
+		{
+			return m_rawAmount;
+		}
+
+		inline float Normalized() const
+		{
+			return m_normalizedAmount;
+		}
+
+		const float ONE_CYCLE_AMOUNT;
```

Comment:
子クラスから設定してもらうようになったので、これは定数では無いですね。
```suggestion
		float m_oneCycleAmount;
```
で良いのでは？

また、アクセスレベルもpublicでは無いのでは？

Reply:
そうします

Reply:
degreeとradian でまた別に作成しました